builder/builder.go

```diff
@@ -443,3 +458,62 @@ func parseImageNameFromArgs(cmdArgs string) string {
 	}
 	return cmdArgs[:idx]
 }
+
+func (b *Builder) createFilesForVolume(ctx context.Context, volMount *volume.VolumeMount) error {
+	var args []string
+	var sb strings.Builder
+	sb.WriteString("mkdir " + volMount.Name)
+	if runtime.GOOS == util.WindowsOS {
+		args = []string{"powershell.exe", "-Command"}
+	} else {
+		args = []string{"/bin/sh", "-c"}
+	}
+	for _, valueMount := range volMount.Values {
+		val := valueMount.Value
+		if valueMount.B64dec {
+			decoded, err := base64.StdEncoding.DecodeString(val)
+			if err != nil {
+				return errors.New("failed to decode Base64 value. please make sure value provided is Base64 encoded")
+			}
+			val = string(decoded)
+		}
+		sb.WriteString(" && echo ")
+		sb.WriteString(val)
```

Comment:
if the decoded value is multi-line with `\n`, will it have any strange bash behavior?

Reply:
Yes you are right this will pose a problem. I will look into fixing it. I also realized that I need to provide a different set of commands entirely for windows since powershell syntax is slightly different.